### ISSUE TITLE
Main nav updates

### ISSUE THREAD

**Issue by r-bartlett-gsa** [GSA/usagov-benefits-eligibility#581]
As a benefits tool user, in order to navigate between USAGov and benefits tool subsite without confusion, I would like the benefit tool subsite to resemble USAGov look and feel as close as possible.

Acceptance criteria:

- [x] Update main nav to match USAGov's beta site (no mega menu)

Resources:

- [Design review document](https://docs.google.com/document/d/1mssKCrkDqNA4GrD-FO22pK5ZifI43hXw88Z8AYvdO4I/edit?usp=sharing) (https://github.com/GSA/usagov-benefits-eligibility/issues/553)
- [Design prioritization spreadsheet](https://docs.google.com/spreadsheets/d/1JxpXDun8nagwVuYwc5a1PSZCPaGzhGcyJvIIYgX396s/edit?usp=sharing)

Definition of Done:
- [x] Code complete
- [x] Tests coverage is greater than team benchmark (90% goal)
- [x] Security scans passed
- [x] Acceptance Criteria is met and it works as expected
- [ ] Accessibility tested
- [ ] Cross browser tested
- [x] Build process and deployment is automated and repeatable 
- [ ] Load testing/performance testing 
- [ ] Self Documentation whenever possible
- [ ] Feature toggles if appropriate 
- [x] Deployed to staging
- [ ] Usability testing
- [ ] PR approved / Peer reviewed
- [ ] A11y tested by Meghan
- [ ] PO approved (UX, usability, edge cases)

More details about each item can be find in [BEARS P4 DoD google doc](https://docs.google.com/document/d/118sdf54uF1vz-5pMvGodV8LRnrUJfjMbHcarCA8AS70/edit?usp=sharing)

[QA checklist](https://docs.google.com/document/d/1AUO-vg0t7DYgQsNouKtz4fFPwelBzZaDZMCkjmPQy0Y/edit?usp=sharing)

**Comment by fpigeonjr** [comment]
This is part of  #579.

**Comment by danny-englander** [comment]
This is dependent on #580 and #582.

**Comment by fpigeonjr** [comment]
I have completed my involvement with this GitHub issue and am stepping away from it. 
Thank you for your understanding.

**Comment by danny-englander** [comment]
@nehemiah-abuga Ready for testing. 

- [ ] Pull `multi-life-event-2023` to your local
- [ ] run `npm i`
- [ ] Run `npm run dev`
- [ ] Go to http://localhost:3000
- [ ] Inspect the main navigation and ensure that the link text and links match what is on https://beta.usa.gov/
- [ ] Switch language to Spanish
- [ ] Inspect the main navigation and ensure that the link text and links match what is on https://beta.usa.gov/es

**Comment by nehemiah-abuga** [comment]
@danny-englander - Beta.usa.gov has Travel while local:3000 has immigration and U.S. citizenship. Please let me know if this is the expected functionality

**Comment by amycole501** [comment]
1. Keyboard testing appeared to be fine.
2. When zooming in 200% on Chrome, the menu button appears after shifting to a mobile layout. The button in Chrome almost appears to be cut off on the right. I'm not sure it's an error per se but it looks almost like it's being covered up by the right scroll bar on my screen. let me know if you need a screen shot.
3. When tabbing into the nav from the top of the page and when listening in JAWS I noticed the order of the elements is off. I am taken from the USA.Gov logo straight into the nav. Ideally the person would go to the right into the Phone number and search area since those appear above the nav.
4. Make sure to note that it's the primary navigation. I didn't hear it announced in JAWS. The USWDS has their site coded with the correct markup to hear "Primary navigation" in screen readers. https://designsystem.digital.gov/components/overview/

**Comment by danny-englander** [comment]
@amycole501 I have addressed all the issues now except item # 3. Ernie said to pass on that as we are going to put all our efforts into V2 for the most part.

**Comment by amycole501** [comment]
In JAWS the language button says it's both a banner region and a navigation region. I'm not sure which it should be but I think it's banner. May warrant a minor fix. 

The header does now read correctly as "header primary navigation region".

**Comment by danny-englander** [comment]
@amycole501 Can you open a new ticket for a followup on the comment above so we can track it?
